some updates

ArbyKay opened the pull request:
some improves in size maps and stuff

Adriwin06 commented:
Thanks for your contribution!

Could you please provide more details on what you changed? It seems like your changes are not based on the latest version of the branch, leading to many conflicts. Since the merge, I have made some enhancements to your fork. It would have been better if you started working from the updated version.

> Removed Character Selection, as well all reference from the struct, it feels unnecesary and makes Menu being too heavy, now all the Common UI Menus fits only ~30 MBs

I noticed you removed the Character Selection. While it might seem unnecessary to you, it is a nice feature. Users who need it can use it, and those who don't can replace it with their own implementation, or don't use it at all: it give the choice to the user.
As for the package size, this isn't a significant concern, as a few megabytes won't make much difference.

ArbyKay commented:
I removed the hard reference Push Menu, and also i removed the selection
character because it also make a hard reference, i think there should be
another way to improve that feature but i wasnt interested in work on that,
i tried to keep update to the recent version but idk how to do that.

El lun, 6 ene 2025 5:42 a. m., Adriwin ***@***.***> escribió:

> Thanks for your contribution!
>
> Could you please provide more details on what you changed? It seems like
> your changes are not based on the latest version of the branch, leading to
> many conflicts. Since the merge, I have made some enhancements to your
> fork. It would have been better if you started working from the updated
> version.
>
> Removed Character Selection, as well all reference from the struct, it
> feels unnecesary and makes Menu being too heavy, now all the Common UI
> Menus fits only ~30 MBs
>
> I noticed you removed the Character Selection. While it might seem
> unnecessary to you, it is a nice feature. Users who need it can use it, and
> those who don't can replace it with their own implementation, or don't use
> it at all: it give the choice to the user.
> As for the package size, this isn't a significant concern, as a few
> megabytes won't make much difference.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Adriwin06/Ultimate-CommonUI-Menu-System/pull/24#issuecomment-2572736621>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2XQJYMXDSUUJKM22TS4QFL2JJFXVAVCNFSM6AAAAABUUOUNIGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNZSG4ZTMNRSGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
